languages/js/src/Polar.ts

```diff
@@ -242,8 +241,12 @@ export class Polar {
    * @param params An optional object with extra parameters.
    */
   registerClass(cls: Class, params?: ClassParams): void {
+    params = params ? params : {};
+    params.id = this.#ffiPolar.newId();
+
     const clsName = this.getHost().cacheClass(cls, params);
     this.registerConstant(cls, clsName);
+    this.getHost().registerMros();
```

Comment:
If loadSources isn't called and we run queries directly we need MROs. This is fine since it's idempotent and perf isn't an issue b/c this is just for Oso instance creation.